components/ScoreCard.vue

```diff
@@ -248,10 +291,18 @@
         <ul>
           <li v-bind:class="{ good: manifestData && manifestData.shortcuts }">
             <div class="listSubDiv">
-              <span class="cardIcon" aria-hidden="true" v-if="manifestData.shortcuts">
+              <span
```

Comment:
Can you instruct vscode not to reformat files? It's hard to see what's actually changed.

Reply:
Yep can do!